testsupport/spacebinding/spacebindingrequest.go

```diff
@@ -34,6 +34,11 @@ func WithNamespace(ns string) SpaceBindingRequestOption {
 func CreateSpaceBindingRequest(t *testing.T, awaitilities wait.Awaitilities, memberName string, opts ...SpaceBindingRequestOption) *toolchainv1alpha1.SpaceBindingRequest {
 	memberAwait, err := awaitilities.Member(memberName)
 	require.NoError(t, err)
+
+	return CreateSpaceBindingRequestOnMember(t, memberAwait, opts...)
+}
+
+func CreateSpaceBindingRequestOnMember(t *testing.T, memberAwait *wait.MemberAwaitility, opts ...SpaceBindingRequestOption) *toolchainv1alpha1.SpaceBindingRequest {
```

Comment:
I think those changes are left overs from when you were using SBRs , but it doesn't harm to have them anyways. So feel free to keep those or drop them.

Reply:
done in 843f770711828865aff04721ba020e9c7762c48e